src/Feature/Account/code/Controllers/AuthenticationController.cs

```diff
@@ -54,20 +54,20 @@ public ActionResult ValidateCredentials(UserLoginModel userLogin)
         }
 
         [HttpPost]
-        [ActionName("signin")]
+        [ActionName("signIn")]
```

Comment:
It's better to leave `signin`, because on FE side we have direct mention of this action in current notation.

Reply:
Fixed spelling in front end side